toolchain/check/keyword_modifier_set.h

```diff
@@ -74,19 +76,19 @@ class KeywordModifierSet {
     return set_ & other.set_;
   }
 
-  // Return a builder that implicitly converts to the specified enumeration type
-  // mapping the cases passed to the `Case`s specified. For example:
+  // Return a builder that returns the new enumeration type once a series of
+  // mapping `Case`s and a final `Default` are provided:
```

Comment:
```suggestion
  // mapping `Case`s and a final `Default` are provided. For example:
```

FWIW I think your previous separate "For example" read better.